src/conv-core/converse.h

```diff
@@ -1104,15 +1104,19 @@ extern void __cmi_assert(const char *);
      (__cmi_assert ("Assertion \"" __CMK_STRING(expr) \
                     "\" failed in file " __FILE__ \
                     " line " __CMK_XSTRING(__LINE__) "."), 0)))
-
-#if ! CMK_ERROR_CHECKING
-#define CmiAssert(expr) ((void) 0)
-#else
-#define CmiAssert(expr) \
+#define CmiEnforceMsg(expr, msg)                    \
```

Comment:
Will this not allow for formatted strings? In a lot of cases then, we'd still probably want to do `if (condition) CkAbort(...)` instead.

Reply:
Ah, yeah, good point, this should support format strings, I'll add support for that.